commands/add.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs')
+let tasks = JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
```

Comment:
yes. you can modify the `array` or `object` within a constant variable. you just cannot change the variable to point to something else